build/ci/.azure-devops-macos.yml

```diff
@@ -53,7 +46,7 @@ jobs:
       msbuildLocationMethod: version
       msbuildVersion: latest
       msbuildArchitecture: x86
-      msbuildArguments: /r /p:Configuration=Release /detailedsummary /bl:$(build.artifactstagingdirectory)/build-macos.binlog
+      msbuildArguments: /r /p:Configuration=Debug /p:EnableCodeSigning=false /p:UseHardenedRuntime=false /detailedsummary /bl:$(build.artifactstagingdirectory)/build-macos.binlog
```

Comment:
That (both flags) looks problematic.

That's the kind of changes that will build just fine but will create issues at runtime.